library/src/main/java/com/google/maps/android/clustering/ClusterManager.java

```diff
@@ -156,11 +162,12 @@ public void setAnimation(boolean animate) {
      * {@link #cluster()} for the map to be cleared.
      */
     public void clearItems() {
-        mAlgorithm.lock();
+        Algorithm<T> algorithm = getAlgorithm();
```

Comment:
Should these local declarations of `Algorithm` be declared `final`, to be safe?

Reply:
Yeah, good call. That makes it more explicit they should remain constant through the entire method execution.

Reply:
Done https://github.com/googlemaps/android-maps-utils/pull/761/commits/ee6d12d402591ba5daf53812dc576ae4c4c1aaee